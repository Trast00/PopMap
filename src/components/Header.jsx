import React from 'react'
import { IoChevronBackOutline } from "react-icons/io5";


function Header(props) {
  const {name, canBack} = props
  return (
    <header className='flex-center'>
      {canBack && (<IoChevronBackOutline />)}
      <p>{name}</p>
      <div></div>
    </header>
  )
}

export default Header
