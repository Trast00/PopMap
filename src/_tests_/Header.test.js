import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import React from 'react';
import Header from '../components/Header.jsx';

describe('test render', () => {
  test('List Rocket should match snapshoot', () => {
    const tree = renderer.create(
      <React.StrictMode>
        <Header name="Home" canBack={false} />
      </React.StrictMode>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
})