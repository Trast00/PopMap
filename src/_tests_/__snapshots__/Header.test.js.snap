// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test render Header should match snapshoot 1`] = `
<header
  className="flex-center"
>
  <p>
    Home
  </p>
  <div />
</header>
`;
